tabs/onboard_logging.js

```diff
@@ -186,11 +175,13 @@ TABS.onboard_logging.initialize = function (callback) {
         for (var i = 0; i < loggingRates.length; i++) {
         	var loggingRate = Math.round(pidRate / loggingRates[i].denom);
         	var loggingRateUnit = " Hz";
-        	if (gcd(loggingRate, 1000)==1000) {
-        		loggingRate /= 1000;
-        		loggingRateUnit = " KHz";
-        		
-        	}
+        	
+            if (loggingRate != Infinity) {
```

Comment:
I'm not sure actually that will be triggered prior to this PR since previously this function wasn't called until all MSP results were received. Now it can actually be run prior to that result and then it sometime ended up with infinity for this value. 
It also really didnt notice that much, just some annoying warning in the inspector. 

I can make a separate PR for this.